docs/scos/dev/guidelines/testing-guidelines/executing-tests/test-the-glue-backend-api.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Test the Glue Backend API
+description: How to test the Glue Backend API
+template: howto-guide-template
+---
+
+This document describes how to set up and run Glue Backend API end-to-end tests. All code references the `ModuleRestApi` module example and the `Pyz` project namespace. When you set up and run the tests for a different project namespace or module, adjust the names accordingly.
+
+## Prerequisites
+
+Make sure the following prerequisites are met:
+
+1. Spryker Testify version 3.52.0 or later is installed.
+- To verify the installation status and version of Spryker Testify, run the following command:
+  ```bash
+  composer info spryker/testify
+  ```
+- To install Spryker Testify, run the following command:
+  ```bash
+  composer require --dev spryker/testify:"^3.52.0"
+  ```
+- To update Spryker Testify, run the following command:
+  ```bash
+  composer update spryker/testify:"^3.52.0"
+  ```
+
+2. To validate the response body against OpenAPI schema, you need to generate the schema first. For details, see [Document Glue API Endpoints](/docs/scos/dev/glue-api-guides/{{site.version}}/document-glue-api-endpoints.html).
+
+## Testing the Glue Backend API endpoint
+
+### 1. Add required configuration to your project `config_default.php` file
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Testify\TestifyConstants;
+
+if (class_exists(TestifyConstants::class)) {
+    $sprykerGlueBackendHost = getenv('SPRYKER_GLUE_BACKEND_HOST');
+    $sprykerGlueBackendPort = (int)(getenv('SPRYKER_GLUE_BACKEND_PORT')) ?: 443;
+    
+    $config[TestifyConstants::GLUE_BACKEND_API_DOMAIN] = sprintf(
+        'https://%s%s',
+        $sprykerGlueBackendHost,
+        $sprykerGlueBackendPort !== 443 ? ':' . $sprykerGlueBackendPort : '',
+    );
+}
+```
+
+### 2. Prepare a codeception test files
+
+#### 1. Create the `codeception.yml` with configuration required for your E2E test:
+
+**tests/PyzTest/Glue/ModuleBackend/codeception.yml**
+
+```yaml
+namespace: PyzTest\Glue\ModuleBackend
```

Comment:
* Let's change ModuleBackend => ModuleBackendApi in the document.

Reply:
Discussed, and decided to leave it as is and adjust once we have conventions for backend API tests.